src/components/BookState.js

```diff
@@ -0,0 +1,15 @@
+function BookState() {
+  return (
+    <div>
+      <p>type</p>
+      <h3>Book Name</h3>
+      <p>Author</p>
+      <ul>
+        <li><button type="button">Comments</button></li>
+        <li><button type="button">Remove</button></li>
+        <li><button type="button">Edit</button></li>
+      </ul>
+    </div>
+  );
```

Comment:
- [ Optional ] I think it would be better if the `Book name` and `Author` fields are not hardcoded in the `BookState` component, it would be nice if they are passed as `props` to this component. Please see the example code below. I highly recommend this change since it will be needed in future milestones of this project.

```JavaScript
function BookState({title, author}) {
  return (
    <div>
      <p>type</p>
      <h3>{ title }</h3>
      <p>{ author }</p>
      <ul>
        <li><button type="button">Comments</button></li>
        <li><button type="button">Remove</button></li>
        <li><button type="button">Edit</button></li>
      </ul>
    </div>
  );
}
export default BookState;
```